locale/shine/extensions/ban/ruRU.json

```diff
@@ -0,0 +1,21 @@
+{
+	"ADD_BAN": "Добавить блокировку",
+	"SELECT_PLAYER": "Выбрать игрока.",
+	"DURATION_LABEL": "Длительность блокировки (в минутах, 0 бессрочная блокировка):",
+	"DURATION_HINT": "Задайте длительность блокировки...",
+	"DURATION_PERMANENT": "Длительность: Навсегда.",
+	"DURATION_TIME": "Длительность: {Time}",
+	"REASON": "Причина:",
+	"Bans": "Блокировки",
+	"BANNED_BY": "Заблокирован",
+	"EXPIRY": "Истекает",
+	"NEVER": "Никогда",
+	"DATE_FORMAT": "%д %м %г %ч:%м",
```

Comment:
The date format needs to be the same as English, as this is a special format Lua reads to build a date string. The only thing you might want to change is the ordering.

The English version (`%d %B %Y %H:%M`) results in `23 Jan 2016 19:00`, where `%d` is the day number, `%B` is a short name for the month, `%Y` is the year, `%H` is the hour and `%M` is the minute.